vast/integration/misc/schema/zeek-conn.schema

```diff
@@ -1,7 +1,8 @@
 type port = count
+type timestamp = time
```

Comment:
Why do we need these duplicated here?

Reply:
Schema files that are passed on the command line don't share the same parsing context as the ones that are part of the directory loading process. So these definitions are still required.

Reply:
Can you create a follow-up story to discuss this?